src/main.ts

```diff
@@ -962,3 +985,35 @@ const cleanUpProxies = async () => {
     kill(proc.pid)
   })
 }
+
+const generateUniqueFileName = async ({
```

Comment:
This should be in a util file, but I'd need to create a new one, since `utils/file.ts` has some exports that are used in `render` :/

Reply:
This is where having 

```
shared/
  - utils/
    - file.ts
main/
  - index.ts
render/
  - utils/
    - file.ts
  - index.ts
```

This would allows us to setup a `no-restricted-imports` rule in ESLint so that things in `render/` can't import from `main/` and vice versa.